rest_framework_tus/utils.py

```diff
@@ -2,28 +2,27 @@
 from __future__ import unicode_literals
 
 import os
+import six
 
-import copy
 import tempfile
 import hashlib
 
 from .compat import encode_base64
 
 
-def encode_base64_to_string(string_value):
+def encode_base64_to_string(data):
     """
-    Helper to encode a string or bytes value to a base64 string
+    Helper to encode a string or bytes value to a base64 string as bytes
 
-    :param six.text_types string_value:
-    :return str:
+    :param six.text_types data:
+    :return six.binary_type:
     """
-    data = copy.copy(string_value)
```

Comment:
This piece of code was totally unnecessary. Copy is meant for copying mutable objects to make sure that modifications don't affect the original data. As strings are immutable, this has literally no effect, except some small performance hit.